accessibility-checker-engine/package.json

```diff
@@ -38,7 +38,8 @@
         "ts-node": "^10.5.0",
         "typescript": "^4.5.4",
         "webpack": "^5.65.0",
-        "webpack-cli": "^4.9.1"
+        "webpack-cli": "^4.9.1",
+        "util": "^0.12.5"
```

Comment:
What are we using from util? I think there's a build-in Node util package, so, not sure this is even necessary.

Reply:
removed.  left over from debugging.